src/line.js

```diff
@@ -34,12 +36,17 @@ SVG.Line = SVG.invent({
       return this.attr(this.array().size(p.width, p.height).toLine())
     }
   }
-  
+
   // Add parent method
 , construct: {
     // Create a line element
     line: function(x1, y1, x2, y2) {
-      return this.put(new SVG.Line).plot(x1, y1, x2, y2)
+      var line = this.put(new SVG.Line)
+
+      // make sure plot is called as a setter
+      return (x1 != null) ?
+        line.plot(x1, y1, x2, y2) :
+        line.plot([[0,0], [0,0]])
```

Comment:
Unfortunately, I can't make this change here. I just tested it and it make some tests fails. I think that is because x1 is not necessarily a number, it can also be an array, a string and a `SVG.PointArray`. 

But, I was able to use the boolean operator in the constructor of `SVG.Path`, `SVG.Polyline` and `SVG.Polygon`. So that comment was still useful :wink: .